tools/gen_compilation_database.py

```diff
@@ -21,13 +21,14 @@ def generateCompilationDatabase(args):
   # We need to download all remote outputs for generated source code, we don't care about built
   # binaries so just always strip and use dynamic link to minimize download size.
   bazel_options = shlex.split(os.environ.get("BAZEL_BUILD_OPTIONS", "")) + [
-      "-c", "fastbuild", "--build_tag_filters=-manual",
+      "-c", "fastbuild", "--build_tag_filters=-manual,-nocompdb",
       "--experimental_remote_download_outputs=all", "--strip=always"
   ]
   if args.run_bazel_build:
     runBazelBuildForCompilationDatabase(bazel_options, args.bazel_targets)
 
   subprocess.check_call(["bazel", "build"] + bazel_options + [
+      "--build_tag_filters=-nocompdb",
```

Comment:
Pedantically shouldn't we also exclude manual per above?

Reply:
actually that manual is redundant since it is automatically excluded. and the above option is inherited here so this line is not needed.